### PR TITLE
Use a toggle button to show/hide messages

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/kiegroup/appformer/pull/231
* https://github.com/kiegroup/kie-wb-common/pull/1494
* https://github.com/kiegroup/drools-wb/pull/829
* https://github.com/kiegroup/jbpm-designer/pull/755
* https://github.com/kiegroup/optaplanner-wb/pull/263

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by paulovmr** [comment]
jenkins execute full downstream build

File: kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/compiler/plugin/KieMetadataTest.java [hunk 1]
```diff
@@ -72,6 +73,7 @@ public void setUp() throws Exception {
     }
 
     @Test
+    @Ignore
```
Comment (ederign) [inline kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/compiler/plugin/KieMetadataTest.java]: ?

Comment (paulovmr) [reply kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/compiler/plugin/KieMetadataTest.java]: Fixed.

**Comment by porcelli** [comment]
test failing not related to this issue, other PRs failing too.

**Pull request merged**